arch/x86/kvm/pkvm/init_finalize.c

```diff
@@ -149,35 +149,13 @@ static int create_hyp_mmu(const struct pkvm_mem_info infos[], int nr_infos)
 static int create_host_mmu(const struct pkvm_mem_info infos[], int nr_infos,
```

Comment:
re commit message: AFAICS this patch has no functional change, it only moves code from one function to another, but "pkvm: x86: Map memory blocks and MMIO during host mmu init" sounds like it actually changes the way how it works. So maybe e.g. "pkvm: x86: Move host MMU mapping setup into pkvm_host_mmu_init()"?